Formula/ibex.rb

```diff
@@ -1,8 +1,8 @@
 class Ibex < Formula
   desc "C++ library for constraint processing over real numbers"
   homepage "https://web.archive.org/web/20190826220512/www.ibex-lib.org/"
-  url "https://github.com/ibex-team/ibex-lib/archive/ibex-2.8.8.tar.gz"
-  sha256 "8de5012232009e29b833e68cffbb62be228a863aa57712207b1933199c3c8e11"
+  url "https://github.com/ibex-team/ibex-lib/archive/ibex-2.8.9.tar.gz"
+  sha256 "fee448b3fa3929a50d36231ff2f14e5480a0b82506594861536e3905801a6571"
   license "LGPL-3.0"
```

Comment:
```suggestion
  license "LGPL-3.0-only"
```

Typical file header doesn't say “or later”:

```
/* ============================================================================
 * I B E X - Dimension of expressions (symbols, etc.)
 * ============================================================================
 * Copyright   : Ecole des Mines de Nantes (FRANCE)
 * License     : This program can be distributed under the terms of the GNU LGPL.
 *               See the file COPYING.LESSER.
 *
 * Author(s)   : Gilles Chabert
 * Created     : Dec 13, 2011
 * ---------------------------------------------------------------------------- */
```

Reply:
👍